presidio-analyzer/presidio_analyzer/entity_recognizer.py

```diff
@@ -26,11 +25,8 @@ class EntityRecognizer:
 
     MIN_SCORE = 0
     MAX_SCORE = 1.0
-    CONTEXT_SIMILARITY_THRESHOLD = 0.65
-    CONTEXT_SIMILARITY_FACTOR = 0.35
-    MIN_SCORE_WITH_CONTEXT_SIMILARITY = 0.4
-    CONTEXT_PREFIX_COUNT = 5
-    CONTEXT_SUFFIX_COUNT = 0
+
+    context = []
```

Comment:
already removed prior to review

Reply:
this is still outside the init method

Reply:
do you mean to add MIN_SCORE and MAX_SCORE too into the init function?
I left those intestinally because I didn't think there were going to be different values of 0 and 1.0 for mix max,
do you think otherwise?

Reply:
I meant the confidence variable

Reply:
discussed over teams, moving self.context into init function

Reply:
done